### PR TITLE
created clipboard example app

### PR DESCRIPTION
**Pull request by jennalenoble**
This app uses Clipboard component to copy a csv dataset. It also includes Loading component and a scatter plot that has the ability to change the x variable with RadioItems.

### CODE REVIEW

File: examples/clipboard-csv.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import pandas as pd
+import plotly.express as px
+
+app = Dash(__name__)
+
+df = pd.read_csv("https://raw.githubusercontent.com/plotly/datasets/master/mtcars.csv")
+
+app.layout = html.Div(
+    [
+        html.H2("Csv Clipboard Example"),
```
Comment (Coding-with-Adam) [inline examples/clipboard-csv.py:11]:
```suggestion
        html.H2("CSV Clipboard Example"),
```

File: examples/clipboard-csv.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+)
+def custom_copy(_, data):
+    dff = pd.DataFrame(data)
+    return dff.to_csv(index=False)  # include headers
+
+@app.callback(
+    Output("clipboard-csv-x-graph", "figure"),
+    Input("clipboard-csv-x-selection", "value")
+)
+
```
Comment (Coding-with-Adam) [inline examples/clipboard-csv.py:44]: delete empty space between callback decorator and callback function.

File: examples/clipboard-csv.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        html.P("Copy mtcars.csv by clicking on the clipboard below:"),
+        dcc.Clipboard(id="clipboard-csv-x-clip", style={"fontSize":20}),
+        dash_table.DataTable(
+            df.to_dict("records"),
+            [{"name": i, "id": i} for i in df.columns],
+            id="clipboard-csv-x-table",
+            page_size = 10
+
+        ),
+        dcc.Loading(children = [dcc.Graph(id="clipboard-csv-x-graph")], type="dot"),
```
Comment (Coding-with-Adam) [inline examples/clipboard-csv.py:21]:
```suggestion
        dcc.Graph(id="clipboard-csv-x-graph"),
```

File: examples/clipboard-csv.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        html.P("Select x variable:", style={"textAlign": "center"}),
+        dcc.RadioItems(id="clipboard-csv-x-selection", 
+                       options=['disp', 'hp', 'drat', 'wt', 'qsec'], 
+                       value='hp', 
+                       style={"textAlign": "center"}),
```
Comment (Coding-with-Adam) [inline examples/clipboard-csv.py:26]:
```suggestion
                       style={"textAlign": "center"}
                       inline=True),
```

File: examples/clipboard-csv.py [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+import plotly.express as px
+
+app = Dash(__name__)
+
+df = pd.read_csv("https://raw.githubusercontent.com/plotly/datasets/master/mtcars.csv")
+
+app.layout = html.Div(
+    [
+        html.H2("Csv Clipboard Example"),
+        html.P("Copy mtcars.csv by clicking on the clipboard below:"),
```
Comment (Coding-with-Adam) [inline examples/clipboard-csv.py:12]:
```suggestion
        html.P("Copy mtcars.csv by clicking on the clipboard below. Paste the copied content into a notepad or Excel sheet to see the result."),
```

File: examples/clipboard-csv.py [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+
+@app.callback(
+    Output("clipboard-csv-x-clip", "content"),
+    Input("clipboard-csv-x-clip", "n_clicks"),
+    State("clipboard-csv-x-table", "data"),
+)
+def custom_copy(_, data):
+    dff = pd.DataFrame(data)
+    return dff.to_csv(index=False)  # include headers
```
Comment (Coding-with-Adam) [inline examples/clipboard-csv.py:38]: Regarding the comment, I think `index=False` means "do not include row names".

**Comment by Coding-with-Adam** [comment]
> It might be good to take the `dcc.Loading` out of this example and create another example to demo `dcc.Loading` component. @Coding-with-Adam - what do you think?

Good job @jennalenoble .
Agreed, we can take out dcc.Loading from this example.

**Comment by jennalenoble** [comment]
Thanks so much for the feedback Ann & Adam! The recent commit incorporates the code changes and uses Black formatting in both files.

**Comment by AnnMarieW** [comment]
Both apps look good to me! :woman_dancing:

**Pull request merged**